dev/TwoPaneView/InteractionTests/TwoPaneViewTests.cs

```diff
@@ -245,27 +246,46 @@ public void PaneLengthTest()
 
                 VerifyPaneSize(1, controlWidth / 2.0, 0);
                 VerifyPaneSize(2, controlWidth / 2.0, 0);
-                
+
                 Log.Comment("Verify changing pane 2 length to pixel sizing");
                 SetLength(2, 199, "Pixel");
 
                 VerifyPaneSize(1, controlWidth - 199, 0);
                 VerifyPaneSize(2, 199, 0);
-                
+
                 Log.Comment("Verify column sizes stay the same when switching pane order");
                 SetWideModeConfiguration(WideModeConfiguration.RightLeft);
 
                 VerifyPaneSize(1, controlWidth - 199, 0);
                 VerifyPaneSize(2, 199, 0);
-                
+
                 Log.Comment("Verify lengths apply to top/bottom configuration");
                 SetControlWidth(ControlWidth.Narrow);
-                
+
                 VerifyPaneSize(1, 0, controlHeight - 199);
                 VerifyPaneSize(2, 0, 199);
             }
         }
 
+
+        // Verify that tabbing around puts us where we expect
+        [TestMethod]
+        public void VerifyTabKeyWorks()
+        {
+            using (var setup = new TestSetupHelper("TwoPaneView Tests")) // This clicks the button corresponding to the test page.
+            {
+                UIObject twoPaneView = TryFindElement.ByName("TwoPaneView");
+                Verify.IsNotNull(twoPaneView, "TwoPaneView is present");
+                twoPaneView.SetFocus();
```

Comment:
I get at crash at this line:

Test method Windows.UI.Xaml.Tests.MUXControls.InteractionTests.TwoPaneViewTests.VerifyTabKeyWorks threw exception: 
    System.Runtime.InteropServices.COMException: Error HRESULT E_FAIL has been returned from a call to a COM component.
  Stack Trace: 
    IUIAutomationElement.SetFocus()
    UIObject.SetFocus()
    TwoPaneViewTests.VerifyTabKeyWorks() line 278